code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -748,6 +750,11 @@ var/global/list/robot_modules = list(
 
 /obj/item/robot_module/syndicate/New(var/mob/living/silicon/robot/R)
 	..()
+
+	R.faction = "syndicate" // prevents viscerators from attacking us
+	for(var/datum/robot_component/RC in R)
+		RC.max_damage *= 2
```

Comment:
Not a fan of that, especially in combination with #9123 which already doubles the cyborg component health.

Reply:
yeah that's fine. would you be alright with a x1.5 increase? to show that syndicate borgs are tougher

Reply:
I dont believe that is needed atm with the 2x health buff we arleady have.
Should there really be a need to further buff them in the future, that can be done.
But we should wait a sufficient time to see how the 2x health buff works out.